subsystem/src/main/java/org/wildfly/extension/grpc/GrpcConstants.java

```diff
@@ -0,0 +1,33 @@
+/*
+ *  Copyright 2022 Red Hat
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.wildfly.extension.grpc;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.jboss.as.controller.AttributeDefinition;
+
+public class GrpcConstants {
+
+	public static final String WILDFLY_GRPC_SERVER_PORT = "wildfly-grpc-server-port";
```

Comment:
No tabs please :)

Reply:
These names should be more generic like "server-port". There is no real reason for the "wildfly-grpc".

Reply:
> No tabs please :)

Yes, I was meaning to ask where to find the eclipse format/template files ...

> These names should be more generic like "server-port".

Ok

Reply:
@ronsigal do you use Eclipse, I thought I was the only one left - two of us use Eclipse !!!

I always get my settings from the WildFly Core repo under ide

Reply:
@darranl, I'm thinking back to when Bill Burke referred to Eclipse as "open source caca" ;-)

Just don't make me learn Emacs !!

Reply:
@ronsigal The Eclipse format rules are under [build-config/src/main/resources/etc/eclipse-formatter-config.xml](https://github.com/wildfly-extras/wildfly-grpc-feature-pack/blob/main/build-config/src/main/resources/etc/eclipse-formatter-config.xml)

Reply:
vi, baby!!

I'm guessing I learned vi in 1983, when I got my first Unix account.  I'm reminded of one of my greatest accomplishments in those days, which was spending several hours figuring out how a colleague of mine could send email from Brooklyn College to another researcher at the University of Michigan. You had to map out a sequence of intervening machines. E.g., Brooklyn College -> a machine at NYU -> a machine at University of Pennsylvania ... . Those were the days.  ;-)

Reply:
Tabs, names fixed